content/vault/v1.20.x/content/docs/updates/important-changes.mdx

```diff
@@ -459,4 +459,16 @@ filters you have two options:
 1. Spread them out among the nodes of the Vault cluster.
 1. Only subscribe to events on the active node of the cluster.
 
-@include '../../../global/partials/important-changes/known-issue/multi-seal-rewrap.mdx'
\ No newline at end of file
+@include '../../../global/partials/important-changes/known-issue/multi-seal-rewrap.mdx'
+
+### Rotation timing shifts across vault restarts ((#rotation-drift)) <EnterpriseAlert inline="true" />
+
+| Change      | Affected version | Fixed version
+| ----------- | ---------------- | -------------
+| Known issue | 1.20.0+ent  | 1.20.5+ent
+
+When you set a `rotation_schedule`, Vault interprets it relative to the local time zone
+of the machine. After a restart or leadership change, Vault interprets the `rotation schedule` relative
+to UTC, which may result in shifted rotation times depending on the time zone offset of the Vault instance.
```

Comment:
```suggestion
When you set `rotation_schedule`, Vault interprets the schedule definition
relative to the local time zone of the machine. After a restart or leadership
change, Vault interprets the value of `rotation schedule` relative to UTC, which
may result in shifted rotation times depending on the time zone offset of the
Vault instance.
```
Tweak language to avoid using a parameter name as an English phrase